src/index.html

```diff
@@ -9,7 +9,7 @@
 	<script src="index.js"></script>
 </head>
 <body>
-	<div id="banner" />
+	<div id="banner"></div>
```

Comment:
Sadly `div`s can't be auto-closed. If you check how the old markup was interpreted by the browser (chrome at least) in the inspector, you would see that it actually tried to have the banner div wrap the content div. When the banner div had its contents injected, there's a good chance you might overwrite the content div by accident. Closing the div properly prevents that